cmd/virtual-kubelet/main.go

```diff
@@ -54,17 +55,20 @@ func main() {
 
 	var opts root.Opts
 	optsErr := root.SetDefaultOpts(&opts)
+	opts.Provider = "azure"
```

Comment:
Given this always has to be set, it might be nice to do something like newProviderOpts(providerName), which also sets default opts.

Reply:
We could if there are other options to do as well.